@navikt/core/react/src/link/Link.tsx

```diff
@@ -4,6 +4,16 @@ import { OverridableComponent } from "../util/OverridableComponent";
 
 export interface LinkProps
   extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Whether the link is suitable for being inlined in text.
+   * @default false
+   */
+  inline?: boolean;
```

Comment:
Så Polaris bare bruker `removeUnderline` for denne 🤔 Blir kanskje da mer deklarativt for bruker.

Reply:
Ja, tanken var å ha mer "smart" med å håndtere spacing iht surrounding BodyLong/BodyShort, men uten dette er det ikke mye annet utenom underline. Kanskje greit å være så spesifikk som mulig. Naming blir mer "hva det er" i stedenfor "hvordan vi har tenkt det skal brukes" som er mer intuitivt + mindre begrenset / bias fra vår side... renamer! 🙌

Reply:
eneste er at, man får underline på :hover, så å kalle det "removeUnderline" blir _teknisk_ sett feil... men det kan bli et issue for fremtiden 😂. Mer riktig er noe ord for "underlineInversion" som er motsatt av det vanlige hover vs nohover.